src/components/BriefList/BriefsList.jsx

```diff
@@ -0,0 +1,12 @@
+import {useEffect} from "react";
+import {getBriefList} from "../../services/BriefData.service";
+
+const BriefList = (props) => {
+  useEffect(() => {
+    getBriefList();
+  });
```

Comment:
overall when you use `useEffect` try to always provide the dependency array

Reply:
also you have different indentation in different files

Reply:
The indentation will be fixed after adding the linter